wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -116,26 +118,28 @@ class WireGenerator(
 }
 
 private fun parseFileDescriptor(fileDescriptor: DescriptorProtos.FileDescriptorProto, descs: DescriptorSource): ProtoFileElement {
-  val path = mutableListOf<Int>()
   val location = Location.get(fileDescriptor.name)
```

Comment:
I think this might be unused now. (Sorry, that's my bad: I forgot to remove it in my PR.)

Reply:
It's used on the `ProtoFileElement` construction